packages/dev/core/src/Materials/PBR/pbrBaseMaterial.ts

```diff
@@ -2319,6 +2359,11 @@ export abstract class PBRBaseMaterial extends PushMaterial {
                 }
 
                 ubo.updateFloat("baseWeight", this._baseWeight);
+                if (this._baseDiffuseRoughness !== null && this._baseDiffuseRoughness !== undefined) {
+                    ubo.updateFloat("baseDiffuseRoughness", this._baseDiffuseRoughness);

```

Comment:
ubo.updateFloat("baseDiffuseRoughness", this._baseDiffuseRoughness ?? this._roughness);